src/plugins/bar-graph/bar-graph-content.ts

```diff
@@ -151,28 +159,35 @@ export const BarGraphContentModel = TileContentModel
     }
   }))
   .actions(self => ({
-    setPrimaryAttributeColor(colorIndex: number) {
-      self.primaryAttributeColor = colorIndex;
-    },
-    setSecondaryAttributeKeyColor(key: string, colorIndex: number) {
-      if (!self.secondaryAttribute) return;
+    setAttributeKeyColor(key: string, colorIndex: number, attr: "primary" | "secondary") {
+      const attribute = attr === "primary" ? self.primaryAttribute : self.secondaryAttribute;
+      const colorMap = attr === "primary" ? self.attributeColorMap : self.attributeColorMap;
```

Comment:
In general I think leaving the `primary...` and `secondary...` methods distinct is reasonable, but this line just looks wrong with the same thing before and after the `:`.

Reply:
Whoops. Thanks for catching that. I've fixed it.